tests/e2e/specs/in-game/oneOffs.spec.js

```diff
@@ -42,40 +42,93 @@ describe('Untargeted One-Offs', () => {
     playOutOfTurn('points');
   }); // End ace one-off
 
-  it('Plays a five to draw two cards', () => {
-    // Setup
-    cy.loadGameFixture(0, {
-      // Player is P0
-      p0Hand: [Card.ACE_OF_CLUBS, Card.FIVE_OF_SPADES, Card.FIVE_OF_HEARTS],
-      p0Points: [Card.TEN_OF_SPADES, Card.ACE_OF_SPADES],
-      p0FaceCards: [Card.KING_OF_SPADES],
-      // Opponent is P1
-      p1Hand: [Card.ACE_OF_HEARTS],
-      p1Points: [Card.TEN_OF_HEARTS, Card.ACE_OF_DIAMONDS],
-      p1FaceCards: [Card.KING_OF_HEARTS],
-      // Deck
-      topCard: Card.THREE_OF_CLUBS,
-      secondCard: Card.EIGHT_OF_HEARTS,
-    });
-    // Player plays five
-    cy.playOneOffAndResolveAsPlayer(Card.FIVE_OF_SPADES);
+  describe('Playing FIVES', () => {
+    it('Plays a five to draw two cards', () => {
+      // Setup
+      cy.loadGameFixture(0, {
+        // Player is P0
+        p0Hand: [Card.ACE_OF_CLUBS, Card.FIVE_OF_SPADES, Card.FIVE_OF_HEARTS],
+        p0Points: [Card.TEN_OF_SPADES, Card.ACE_OF_SPADES],
+        p0FaceCards: [Card.KING_OF_SPADES],
+        // Opponent is P1
+        p1Hand: [Card.ACE_OF_HEARTS],
+        p1Points: [Card.TEN_OF_HEARTS, Card.ACE_OF_DIAMONDS],
+        p1FaceCards: [Card.KING_OF_HEARTS],
+        // Deck
+        topCard: Card.THREE_OF_CLUBS,
+        secondCard: Card.EIGHT_OF_HEARTS,
+      });
+      // Player plays five
+      cy.playOneOffAndResolveAsPlayer(Card.FIVE_OF_SPADES);
+  
+      // Assert game state
+      assertGameState(0, {
+        // Player is P0
+        p0Hand: [Card.ACE_OF_CLUBS, Card.FIVE_OF_HEARTS, Card.THREE_OF_CLUBS, Card.EIGHT_OF_HEARTS],
+        p0Points: [Card.TEN_OF_SPADES, Card.ACE_OF_SPADES],
+        p0FaceCards: [Card.KING_OF_SPADES],
+        // Opponent is P1
+        p1Hand: [Card.ACE_OF_HEARTS],
+        p1Points: [Card.TEN_OF_HEARTS, Card.ACE_OF_DIAMONDS],
+        p1FaceCards: [Card.KING_OF_HEARTS],
+        scrap: [Card.FIVE_OF_SPADES],
+      });
+      // Attempt to plays five out of turn
+      cy.get('[data-player-hand-card=5-2]').click(); // five of hearts
+      playOutOfTurn('oneOff');
+    }); // End five one-off
 
-    // Assert game state
-    assertGameState(0, {
-      // Player is P0
-      p0Hand: [Card.ACE_OF_CLUBS, Card.FIVE_OF_HEARTS, Card.THREE_OF_CLUBS, Card.EIGHT_OF_HEARTS],
-      p0Points: [Card.TEN_OF_SPADES, Card.ACE_OF_SPADES],
-      p0FaceCards: [Card.KING_OF_SPADES],
-      // Opponent is P1
-      p1Hand: [Card.ACE_OF_HEARTS],
-      p1Points: [Card.TEN_OF_HEARTS, Card.ACE_OF_DIAMONDS],
-      p1FaceCards: [Card.KING_OF_HEARTS],
-      scrap: [Card.FIVE_OF_SPADES],
+    it('Plays a five to draw one card when at 8 cards in hand', () => {
```

Comment:
We should probably also add test cases for:
1. Playing a 5 with <= 7 cards in hand when there's 2 cards in the deck
2. Playing a 5 with <= 7 cards in hand when there's 1 card in the deck
3. Playing a 5 with 8 cards in hand when there's 2 cards in the deck
4. Playing a 5 with 8 cards in hand when there's 1 card in the deck